themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,182 @@
+
+### What's not great about it?
+
+#### Scaling
+
+ACI supports customizing the amount of resources you can assign to a container, but it will only run one copy. There's no horizontal scaling available, so if you need more resources, you only have one option: scaling vertically.
+
+If you’re running a webserver application, horizontal scaling behind a load balancer might be a better option, so this lack of scaling flexibility can be a real shortcoming.
+
+In addition to this opinionated scaling mechanism, there’s also no auto scaling option for ACI. If you’re finding that you’re hitting the limits of your ACI deployment, you’ll need to manually scale the service.
+
+#### Configuration
+
+ACI has some configuration options missing, which you’d expect would be there from the beginning. One glaring omission is port mapping (for example mapping public port 80 to container port 8080). This omission can mean modifying your container just to get it running in ACI.
+
+ACI’s opinionated architecture seems to have been designed for ease-of-use, which often means more complex workloads just aren’t possible. If you have some unique configuration options, like needing access to privileged container options or complex network requirements, ACI might not be able to support you.
+
+### What we’d use it for
+
+The lack of flexibility in ACI means that some workloads just aren’t able to run there. If you have full control over the container images you’re building and if you just want to get off the ground quickly with minimal fuss and configuration, ACI is a great choice.
+
+Applications with smaller and more predictable usage patterns are a great use case for ACI. The lack of horizontal scaling could be seen as a negative, but from another perspective, it means the use case is straightforward and simple. If you just want to get your application into the hands of your users and think you can accurately predict your usage patterns, choose ACI.
+
+## Azure App Service
+
+Azure App Service is a more fully-featured offering for hosting applications, focusing more on WebApps and API services. It started life as several differing offerings (Azure Websites, Azure Mobile Service, and Azure BizTalk) before being bundled into a single service in March 2015. Its origins as multiple solutions means it has developed tight integrations with other Azure solutions and offers many value-add features depending on your use case.
+
+Because of the myriad features Azure App Service offers, running containers is only a small part of what you can do with it.
+
+With that in mind, let's take a look at where it shines with regard to running containers.
+
+### What’s it good for?
+
+Azure App Services is designed to be a one-stop shop for your web application needs, so it's highly opinionated and pushes you in a certain direction.
+
+#### Integrations
```

Comment:
By contrast with ACI, App Service has full-featured integration with Azure networking. You can run it fully locked out in a virtual network, which is huge for enterprise scenarios.